### PR TITLE
CMC-450-Kustomize_CMC_ClaimStore

### PR DESCRIPTION
**Pull request by rmurphy-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-450

### Change description ###
Migration of application Money-Claims Claim-Store in money-claims namespaces to Kustomize. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/namespaces/money-claims/cmc-claim-store/aat.yaml [hunk 1]
```diff
@@ -4,19 +4,12 @@ metadata:
   name: cmc-claim-store
   namespace: money-claims
 spec:
-  releaseName: cmc-claim-store
-  rollback:
-    enable: true
-    retry: true
   chart:
     git: git@github.com:hmcts/hmcts-charts
     ref: master
```
Comment (reespozzi) [inline k8s/namespaces/money-claims/cmc-claim-store/aat.yaml:9]: Already referenced in base chart

File: k8s/namespaces/money-claims/cmc-claim-store/demo.yaml [hunk 2]
```diff
@@ -4,18 +4,12 @@ metadata:
...
 spec:
-  releaseName: cmc-claim-store
-  rollback:
-    enable: true
-    retry: true
   chart:
     git: git@github.com:hmcts/hmcts-charts
     ref: master
     path: stable/cmc-claim-store
   values:
```
Comment (reespozzi) [inline k8s/namespaces/money-claims/cmc-claim-store/demo.yaml:11]: Same here and likely in the other {env}.yaml files that don't need a specific version

File: k8s/namespaces/money-claims/cmc-claim-store/perftest.yaml [hunk 3]
```diff
@@ -67,10 +61,7 @@ spec:
         MILO_CSV_SENDER: noreply@mail-money-claims-nonprod.platform.hmcts.net
         RPA_NOTIFICATIONS_BREATHINGSPACERECIPIENT: civilmoneyclaimsdemo@gmail.com
```
Comment (reespozzi) [inline k8s/namespaces/money-claims/cmc-claim-store/perftest.yaml:62]: Definitely using demo email?

**Review by reespozzi — CHANGES_REQUESTED** [review]

File: k8s/namespaces/money-claims/cmc-claim-store/demo.yaml (hunk 2 above)
Comment (rmurphy-hmcts) [reply k8s/namespaces/money-claims/cmc-claim-store/demo.yaml:11]: Chart section removed from base chart due to different environments using different sources

File: k8s/namespaces/money-claims/cmc-claim-store/aat.yaml (hunk 1 above)
Comment (rmurphy-hmcts) [reply k8s/namespaces/money-claims/cmc-claim-store/aat.yaml:9]: Chart section removed from base chart due to different environments using different sources

File: k8s/namespaces/money-claims/cmc-claim-store/perftest.yaml (hunk 3 above)
Comment (rmurphy-hmcts) [reply k8s/namespaces/money-claims/cmc-claim-store/perftest.yaml:62]: It is currently set to demo.
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/perftest/common/money-claims/claim-store.yaml

**Review by reespozzi — APPROVED** [review]

**Review by akriti-hmcts — DISMISSED** [review]

**Review by dharmendrak — APPROVED** [review]

**Review by robertmarks — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
Looks OK to me if pipeline is OK

**Pull request merged**